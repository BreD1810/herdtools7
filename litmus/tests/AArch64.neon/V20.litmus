AArch64 V20
(* Tests MOV to general *)
{
}

 P0                     ;
 MOVI V0.2S,#1  ;
 MOV W0,V0.S[0] ;
 MOV W1,V0.S[1] ;
 forall (0:X0=1 /\ 0:X1=1)
