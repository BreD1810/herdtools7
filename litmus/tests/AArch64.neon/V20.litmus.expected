Test V20 Required
Histogram (1 states)
1000000:>0:X0=1; 0:X1=1;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (0:X0=1 /\ 0:X1=1) is validated
Hash=9757abe712a4d6d4faf4becce89c26a6
Observation V20 Always 1000000 0
